src/main/java/hw7/pages/JDIBasePage.java

```diff
@@ -0,0 +1,32 @@
+package hw7.pages;
+
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+import hw7.entities.User;
+import hw7.forms.JDILogin;
+import org.openqa.selenium.WebElement;
+
+public class JDIBasePage extends WebPage {
+
+    private JDILogin loginForm;
+
+    @FindBy(id = "user-icon")
+    private Button userButton;
+
+    @FindBy(id = "user-name")
+    private Text username;
+
+
+    public void login(User user) {
+        userButton.click();
+        loginForm.login(user);
+    }
+
+    public boolean checkUsername(User user){
```

Comment:
You should not named a methods by "check" prefix in case if it doesn't have an assert call.
Basically, "boolean" methods have a prefix "is"